test/Concurrency/Runtime/actor_counters.swift

```diff
@@ -4,9 +4,6 @@
 // REQUIRES: concurrency
 // REQUIRES: libdispatch
 
-// Remove with rdar://problem/72439642
-// UNSUPPORTED: asan
-
```

Comment:
This test was disabled under ASan, but really only worked "by accident" in non-ASan configurations.